coalib/bearlib/aspects/collections.py

```diff
@@ -16,6 +16,13 @@ def __init__(self, seq=()):
         list.__init__(self, seq)
 
     def __contains__(self, aspect):
+        if 'bear' in dir(self) and isinstance(aspect, aspectbase):
+            compatible = False
+            for language in self.bear.LANGUAGES:
+                if str(aspect.language).split()[0] == language:
```

Comment:
Or instead of this we could maybe wait for https://github.com/coala/coala/pull/4109